packages/block-library/src/navigation/index.php

```diff
@@ -157,13 +157,19 @@ function render_block_core_navigation( $content, $block ) {
 		return '';
 	}
 
+	$orientation_class_name = 'is-horizontal';
```

Comment:
It doesn't look like `is-horizontal` is added in the `edit.js` file for the block. 

It should ideally be consistent. The class itself seems to be unused, so I don't think it should be added here:
```suggestion
	$orientation_class_name = '';
```